src/plugins/example-blocks/package.json

```diff
@@ -7,14 +7,12 @@
 	"main": "build/index.js",
 	"scripts": {
 		"build": "./scripts/build-blocks.sh",
-		"build:block": "wp-scripts build blocks/$npm_config_block/src/index.js --output-path=blocks/$npm_config_block/build/",
 		"format": "wp-scripts format",
 		"lint:css": "wp-scripts lint-style",
 		"lint:js": "wp-scripts lint-js",
 		"packages-update": "wp-scripts packages-update",
 		"plugin-zip": "wp-scripts plugin-zip",
-		"start": "./scripts/start-blocks.sh",
-		"start:block": "wp-scripts start blocks/$npm_config_block/src/index.js --output-path=blocks/$npm_config_block/build/"
```

Comment:
Do we think we would need these `start:block` and `build:block` scripts for any reason? I removed them because it seemed like maybe not, and also they didn't work quite right 🙃 

If we do want them, I'm happy to put them back and resolve the issues!

Reply:
I think we're good without them, the shell script solution works pretty well as an alternative!